### ISSUE TITLE
`"space-before-combinator": 1` disturb indentation rule...

### ISSUE THREAD

**Issue by Soten-Bluesky** [csscomb/csscomb.js#551]
Hi I want to add a space before combinator, but I want reshape that new line first-letter is not space.
When line break is applied by csscomb, the setting "space-before-combinator: 1" is applied to the new line (first letter is combinator) as well, so the indentation does not become what I intended.

Is there a way to solve this issue?
　
　
# Example:

## Default
```
.foo {
> .foo {
prop: val;
}
}
```

## After using CSSComb:

**Actual result 1（bad）:**
```
space-before-combinator：" ",
```
![bad1](https://user-images.githubusercontent.com/6637993/32031328-73139294-ba3b-11e7-8a90-aa7d0c315f40.jpg)

**Actual result 2（bad）:**
```
space-before-combinator："",
```
![bad2](https://user-images.githubusercontent.com/6637993/32031329-7337d67c-ba3b-11e7-9733-59818f556fb6.jpg)

**The result I am expecting to get（Good）:**
```
space-before-combinator：" ",
```
![good](https://user-images.githubusercontent.com/6637993/32031361-b6c9d674-ba3b-11e7-95e2-3d6209d45a11.jpg)

**Comment by mrleblanc101** [comment]
Bump

**Comment by mariovalney** [comment]
A PR \o/

**Comment by tonyganch** [comment]
Fixed with #599.

**Comment by lhtdesignde** [comment]
Is this released already? I just updated to version 4.3 and I'm seeing exactly this behavior.

**Comment by jdalton** [comment]
@lhtdesignde 
> Is this released already? I just updated to version 4.3 and I'm seeing exactly this behavior.

Yes. Would you please open a new issue with your repo.
